dependency-check-maven-config.xml

```diff
@@ -1,6 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://www.owasp.org/index.php/OWASP_Dependency_Check_Suppression">
 
+    <!-- ddf kernel -->
+    <suppress>
+        <notes>
+            Karaf 4.1.1 uses pax-web-jsp 6.0.3 which is a dependency of ddf-kernel and contains
+            several CVEs: CVE-2016-8735, CVE-2014-0230, CVE-2013-2185, CVE-2011-3190, CVE-2009-3548
```

Comment:
Also, you may want to leave a comment somewhere that this CVE should be reinvestigated after https://codice.atlassian.net/browse/DDF-3021